docs/en/faq/how-kotlin-jdsl-is-different-from-jooq-and-querydsl.md

```diff
@@ -0,0 +1,34 @@
+# How is Kotlin JDSL different from jOOQ and QueryDSL?
+
+Unlike jOOq and QueryDSL, Kotlin JDSL does not generate a `Metadata Model (code generation) for writing queries.
+
+If you use code generation, you'll have to do the following.
+
+```
+Modify tables / entities →
+Compilation errors due to code and metacode mismatch →
+Temporarily change all code associated with the modified code, e.g. by commenting it out, to make it compile.
+Regenerate code
+Remove the temporary code that was processed to pass compilation and replace it with working code.
+```
```

Comment:
Commenting is not required. Depending on how you run your Gradle task, you can generate metadata without commenting.

So I think it would be better to write something like this. 

Modify table and entities -> Entity modification causes compile error -> Run Maven or Gradle task to regenerate metadata -> Write query based on modified entity

This way we have to run a Maven or Gradle test to modify the entity and use the modified entity. This causes an interruption in development, which can interrupt your thoughts.

Reply:
That's something I was dissatisfied with when I wrote it too, I guess I didn't have an example of the essentials. Great, I've incorporated it.